### PR TITLE
translations PR 20180718

### PR DESCRIPTION
**Pull request by enahum**
Ideally this PR should be review by community members for each language.

Note: In the event that you see an english translation instead of the desired language is because this PR was submitted before the translation was made on the Mattermost Translation Server and probably will be submitted for the next PR.

Webapp PR: https://github.com/mattermost/mattermost-webapp/pull/1457

### CODE REVIEW

**Comment by jasonblais** [comment]
@enahum Build failed

**Comment by enahum** [comment]
@jasonblais no idea why this is failing, but it does not look to be related to this PR.

Also I cannot seem to find the reason of the failure in the Jenkins log

**Comment by enahum** [comment]
@jasonblais yes it seems to be a gremlin in the build, so not related to this PR, this is now passing

File: i18n/es.json [hunk 1]
```diff
@@ -2622,6 +2630,18 @@
     "id": "app.import.validate_direct_post_import_data.user_missing.error",
     "translation": "Falta la propiedad obligatoria del mensaje del canal directo: user"
   },
+  {
+    "id": "app.import.validate_emoji_import_data.empty.error",
+    "translation": "Datos de importación del emoticon vacío."
+  },
+  {
+    "id": "app.import.validate_emoji_import_data.image_missing.error",
+    "translation": "El campo de imagén del emoticon en la importación está faltante o vacío."
```
Comment (jespino) [inline i18n/es.json]: This sounds weird in Spanish from Spain. I would use here `El campo de imagén del emoticon en la importación está ausente o vacío.`.

File: i18n/es.json [hunk 2]
```diff
@@ -2622,6 +2630,18 @@
...
+  },
+  {
+    "id": "app.import.validate_emoji_import_data.name_missing.error",
+    "translation": "El campo del nombre del emoticon en la importación está faltante o vacío."
```
Comment (jespino) [inline i18n/es.json]: The same here.

File: i18n/es.json (hunk 1 above)
Comment (enahum) [reply i18n/es.json]: can you please go to the translation server and make the changes there ;)

thanks my friend.

Comment (jespino) [reply i18n/es.json]: Sure

Comment (jespino) [reply i18n/es.json]: Done :)

**Pull request merged**